docs/deployment/operator.md

```diff
@@ -36,14 +36,16 @@ deployment.apps/nsp-trench-a    1/1     1            1           1m
 
 ### Attractor
 
-An attractor resource is responsible for the creation of NSE-VLAN and stateless-lb-frontend. The resources created will be suffixed with either attractor or trench's name.
+Responsible for cluster breakout and attraction of external traffic. The cluster breakout is achieved through a secondary inteface whose setup is governed by the Interface Specification. By default NSM is used to take care of both the inteface management and IP address allocation. Alternatively, Multus can be employed by supplying the name and kubernetes namespace of a Network Attachment Definiton for the Interface Specification.
+
+An attractor resource is responsible for the creation of stateless-lb-frontend, and if the Interface Type is `nsm-vlan` for the creation of NSE-VLAN. The resources created will be suffixed with either attractor or trench's name.
```

Comment:
"...either attractor or trench's name".   based on what?

Reply:
Currently it seems to be only the attractor name that is considered. Could have been the multi-conduit support that changed it...